packages/ketcher-react/src/script/editor/Editor.ts

```diff
@@ -557,6 +557,7 @@ class Editor implements KetcherEditor {
   }
 
   undo() {
+    const ketcherProviderEditor = ketcherProvider.getKetcher().changeEvent;
```

Comment:
It is rather event instance than editor instance, I'd change variable name

Reply:
Maby ketcherChangeEvent?